### PR TITLE
evaluation metrics implementation: Confusion Metrics/ CohenKappa

### PR DESCRIPTION
**Pull request by anda-ren**
## Description

- evaluation metrics collector implementation: Confusion Metrics/ CohenKappa
- fix some code smell issue

## Modules
- [ ] UI
- [X] Controller
- [ ] Agent
- [ ] Client
- [ ] Python-SDK
- [ ] Others

## Checklist
- [x] run code format and lint check
- [x] add unit test
- [x] add necessary doc

### CODE REVIEW

File: server/controller/src/main/java/ai/starwhale/mlops/exception/SWValidationException.java [hunk 1]
```diff
@@ -13,8 +13,8 @@
 
     static final String PREFIX_TIP="invalid request on subject ";
 
-    private String code;
-    private String tip;
+    private final String code;
+    private final String tip;
```
Comment (goldenxinxing) [inline server/controller/src/main/java/ai/starwhale/mlops/exception/SWValidationException.java:17]: now wo have several similar name：“message”、“tip”、“msg”，use a generic name？

Comment (anda-ren) [reply server/controller/src/main/java/ai/starwhale/mlops/exception/SWValidationException.java:17]: "tip" is considered as a user friendly message. "message" and "msg" are more generic than "tip"

**Review by goldenxinxing — DISMISSED** [review]
LGTM

**Comment by tianweidut** [comment]
@anda-ren  plz solve conflicting files

**Review by goldenxinxing — APPROVED** [review]
LGTM

**Pull request merged**